ground/gcs/src/inc/widgetnames.h

```diff
@@ -1,6 +1,18 @@
 #ifndef WIDGETNAMES_H
 #define WIDGETNAMES_H
 
-
+#define CONFIGGADGETWIDGET_SIDEBAR						"configGadgetSideBar"
+#define CONFIGGADGETWIDGET_SIDEBAR_VBOXLAYOUT			"configGadgetSideBarVBox"
+#define CONFIGGADGETWIDGET_HWSETTINGS					"configGadgetHwSettings"
+#define CONFIGGADGETWIDGET_VEHICLETYPE					"configGadgetVehicleType"
+#define CONFIGGADGETWIDGET_INPUT						"configGadgetInput"
+#define CONFIGGADGETWIDGET_OUTPUT						"configGadgetOutput"
+#define CONFIGGADGETWIDGET_ATTITUDE						"configGadgetAttitude"
+#define CONFIGGADGETWIDGET_STABILIZATION				"configGadgetStabilization"
+#define CONFIGGADGETWIDGET_MODULES						"configGadgetModules"
+#define CONFIGGADGETWIDGET_AUTOTUNE						"configGadgetAutotune"
+#define CONFIGGADGETWIDGET_CAMERASTAB					"configGadgetCameraStab"
+#define CONFIGGADGETWIDGET_TXPID						"configGadgetTxPID"
+#define CONFIGGADGETWIDGET_OSD							"configGadgetCameraOSD"
 
```

Comment:
The problem is that you can't use the #defines in the stylesheet because css files don't go through the c pre-processor, and they certainly don't see c++ constants. 😛 I appreciate the idea but I think it's very unlikely these will ever be renamed given that they haven't even been named at any point in the past 7 years! 😀 As you say, naming QObjects is useful because [they come through in usage data](https://github.com/d-ronin/dRonin/blob/next/ground/gcs/src/plugins/usagestatsgadget/usagestatsplugin.cpp#L199).